main.rb

```diff
@@ -1,4 +1,4 @@
-require './book'
+# require './data'
```

Comment:
- [ ] it would be better to remove the commented code in order to keep the file clean and pro